src/customers/name-address-options.md

```diff
@@ -2,33 +2,149 @@
 title: Name and Address Options
 ---
 
-The Name and Address Options determine which attributes are included in the names and addresses that appear in the [customer account]({% link customers/account-dashboard.md %}).
+The Name and Address Options determine which fields are included in the name and address forms when customers create an [account]({% link customers/account-create.md %}) with your store.
 
-## To configure the customer name and address:
+<!--{% if "Default.EE-B2B" contains site.edition %}-->
+![]({% link images/images-ee/storefront-customer-account-address-book.png %}){: .zoom}
+_[Customer Account Sign Up Form]({% link configuration/customers/customer-configuration.md %})_
+
+## Step 1: Set the scope of the configuration
 
 1. On the _Admin_ sidebar, go to **Stores** > _Setting_ > **Configuration**.
 
 1. In the panel on the left, expand **Customers** and choose **Customer Configuration**.
 
 1. Expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **Name and Address Options** section.
 
-    ![]({% link images/images/config-customers-customer-configuration-name-address.png %}){: .zoom}
-    _Name and Address Options_
+   Notice that the scope of the name and address options applies at the `website` level.
+
+1. Scroll up to the top of the page. Then in the upper-left corner, set the scope of the configuration to one of the following:
+
+   - Default Config
+   - Main Website (or specific site for multi-site installations)
+
+   The Name and Address Options section doesn't appear when the scope is set to `Default Store View`.
+
+   ![]({% link images/images-ee/config-customer-configuration-scope.png %}){: .zoom}
+   _[Configuration Scope]({% link configuration/scope.md %})_
+
+## Step 2: Configure the name and address options
+
+1. Return to the **Name and Address Options** section of the Customer Configuration page.
```

Comment:
just above this, name and address option is not bolded as a section name.  check for consistency

Reply:
Fixed.